src/pages/404.tsx

```diff
@@ -16,7 +17,7 @@ export default function NotFound() {
             <Heading as="h1">Page not found</Heading>
             <Flex variant="column" gap={0}>
               <Text variant="lead" className={styles.text}>
-                Sorry! We couldn’t find the page you were looking for.
+                Sorry! We couldn't find the page you were looking for.
```

Comment:
Was this some sort of automatic editor change?
```suggestion
                Sorry! We couldn’t find the page you were looking for.
```

Reply:
No, I just updated to be a standard quote. Can revert this change though.

Reply:
It was a hyphen (which is grammatically correct), this: `'` is a tick, used for feet/inches